dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -1584,6 +1584,20 @@ def get_record_with_sql(select_attribute, table_name, selector)
     assert_removes_field_from_form_geos :longitude
   end
 
+  #
+  # Table: dashboard.ai_tutor_interactions
+  #
+
+  test "deletes all of a soft-deleted user's ai tutor interactions (chat messages)" do
+    student = create :student_with_ai_tutor_access
+    num_ai_tutor_interactions = 3
+    create_list :ai_tutor_interaction, num_ai_tutor_interactions, user: student
+
+    assert_changes -> {AiTutorInteraction.where(user: student).count}, from: num_ai_tutor_interactions, to: 0 do
+      student.destroy
```

Comment:
I believe that `User#destroy` doesn't trigger the `DeleteAccountsHelper#purge_user` which calls `DeleteAccountsHelper#delete_ai_tutor_interactions`, so it won't work